### DISCUSSION TITLE
nested model key name "sort"

### DISCUSSION THREAD

**Discussion by infohojin** [livewire/livewire#4479] [1. Help]
Hi, i write blade template like this
```
<div>
        <input wire:model="forms.sort">
</div>
```

livewire automatically input text like this.

```
function sort() { [native code] }
```

sort array key name is not possible. how can i use this?

**Comment by joshhanley** [comment]
@infohojin I'm not sure what you mean, can you explain in more detail what you mean?

Thanks!

**Comment by JurjenRoels** [comment]
I have no answer. But the same issue.

I have a field name that is called sort. When this is filled with wire:model="modelname.sort" you get the  error @infohojin describes.

@infohojin Did you solve this issue?

Regards
Jurjen
